### PR TITLE
Add new icons

### PR DESCRIPTION
**Pull request by kristen**
#74 
Here is my initial stab at adding icons. We will definitely need to change some, but I didn't want to get too carried away and have merging issues with you guys.

### CODE REVIEW

File: Mapstagram/ViewControllers/GridViewController.swift [hunk 1]
```diff
@@ -21,7 +21,8 @@ class GridViewController: MediaViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "search-icon"), style: UIBarButtonItemStyle.Plain, target: self, action: "didClickSearch")
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .Search, target: self, action: "didClickSearch")
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/GridViewController.swift]: Can these declarations for the search button in map/grid/list be moved to the parent class so it's only in 1 place?  Does the target action pattern still work?

File: Mapstagram/ViewControllers/ListViewController.swift [hunk 2]
```diff
@@ -96,6 +96,7 @@ extension ListViewController: UITableViewDataSource, UITableViewDelegate {
             
             let cell = tableView.dequeueReusableCellWithIdentifier("CommentCell") as CommentCell
             cell.comment = comment
+            cell.first = index == TableViewCells.Comment.rawValue
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/ListViewController.swift]: What's this for?

File: Mapstagram/ViewControllers/ListViewController.swift [hunk 3]
```diff
@@ -30,7 +30,7 @@ class ListViewController: MediaViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "search-icon"), style: UIBarButtonItemStyle.Plain, target: self, action: "didClickSearch")
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .Search, target: self, action: "didClickSearch")
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/ListViewController.swift]: See previous comment

File: Mapstagram/ViewControllers/MapViewController.swift [hunk 4]
```diff
@@ -41,8 +53,9 @@ class MapViewController: MediaViewController {
         // inset adjustments yourself, such as when there is more than one scroll view in the view hierarchy.
         // We have a map view and collection view, so I had to set inset adjustments myself.
         automaticallyAdjustsScrollViewInsets = false
-        
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "search-icon"), style: UIBarButtonItemStyle.Plain, target: self, action: "didClickSearch")
+
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .Refresh, target: self, action: "onRefreshMapButton")
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .Search, target: self, action: "didClickSearch")
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/MapViewController.swift]: See previous comment

File: Mapstagram/ViewControllers/ProfileViewController.swift [hunk 5]
```diff
@@ -79,7 +79,7 @@ class ProfileViewController: UIViewController {
         
         // logout button only applies when viewing authenticated user's profile
         if self.ownProfile {
-            navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Logout", style: UIBarButtonItemStyle.Bordered, target: self, action: "logout")
+            navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "logout"), style: UIBarButtonItemStyle.Bordered, target: self, action: "logout")
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/ProfileViewController.swift]: I'm surprised iOS doesn't have a default logout button.  Oh well.

File: Mapstagram/ViewControllers/ListViewController.swift (hunk 2 above)
Comment (jamietsao) [inline Mapstagram/ViewControllers/ListViewController.swift]: Nevermind, I see what it's for now (further down in PR).

File: Mapstagram/Views/CommentCell.swift [hunk 6]
```diff
@@ -23,6 +27,16 @@ class CommentCell: UITableViewCell {
         }
     }
     
+    var first: Bool! {
+        didSet {
```
Comment (jamietsao) [inline Mapstagram/Views/CommentCell.swift]: :+1:

**Comment by jamietsao** [comment]
One minor refactor comment.  Looks great, merge it!  Can't wait to see the icons!

File: Mapstagram/ViewControllers/GridViewController.swift (hunk 1 above)
Comment (kristen) [inline Mapstagram/ViewControllers/GridViewController.swift]: done :smile:

Comment (jamietsao) [inline Mapstagram/ViewControllers/GridViewController.swift]: Nice!

**Pull request merged**